### PR TITLE
libraw: use upstream cmake build script + fix component names

### PR DESCRIPTION
**Pull request by madebr**
libraw: make libraw buildable with libjpeg-turbo

Specify library name and version:  **libraw/all**

- [X] Depends on  #2733
- [x] Depends on  #2884

---

- [X] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [X] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by uilianries — DISMISSED** [review]
Great job! New options and official cmake support.

File: recipes/libraw/all/conanfile.py [hunk 1]
```diff
@@ -1,62 +1,149 @@
...
+        "with_zlib": [True, False],
+        "with_lcms": [True, False],
+        "with_jasper": [True, False],
+        "with_openmp": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "rawspeed": False,
+        "with_libjpeg": "libjpeg-turbo",
```
Comment (madebr) [inline recipes/libraw/all/conanfile.py:31]:
```suggestion
        "with_libjpeg": "libjpeg",
```

**Review by uilianries — DISMISSED** [review]

**Comment by madebr** [comment]
Depends on #2731

**Comment by madebr** [comment]
@uilianries 
Something is off with the jasper on Macos.
How can I see the content of the jasper packages on macos?
https://conan.io/center/jasper/2.0.19/?user=_&channel=_&revision=&os=Macos&tab=config

**Comment by madebr** [comment]
This needs https://github.com/conan-io/conan-center-index/pull/2733, which is not available on bintray yet.

**Comment by madebr** [comment]
[Portal - 'Still Alive'](http://www.youtube.com/watch?v=Y6ljFaKRTrI "Portal - 'Still Alive'")

**Pull request closed without merging**